examples/monitor/match_bets_with_langfuse_traces.py

```
@@ -212,27 +264,45 @@
 
+        simulations_df = pd.DataFrame.from_records(simulations)
+        simulations_df.sort_values(by="bet_profit", ascending=False, inplace=True)
         overall_md += (
             f"\n\n## {agent_name}\n\n{len(bets_with_traces)} bets\n\n"
-            + pd.DataFrame.from_records(simulations).to_markdown(index=False)
+            + simulations_df.to_markdown(index=False)
         )
+        # export details per agent
+        pd.DataFrame.from_records(details).to_csv(f"{agent_name}_details.csv")
+
+    print(f"Correlation between p_yes mse and total profit:")
```

Ruff

f-string without any placeholders

Remove extraneous `f` prefix

(F541)